arcgis-ios-sdk-samples/Geometry/Format coordinates/FormatCoordinates.storyboard

```diff
@@ -1,168 +1,37 @@
-                                                    </stackView>
-                                                </subviews>
-                                                <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="calibratedWhite"/>
-                                                <constraints>
-                                                    <constraint firstItem="iDJ-b0-uIo" firstAttribute="leading" secondItem="f0e-3N-1ay" secondAttribute="leading" constant="8" id="3MO-Um-VNp"/>
-                                                    <constraint firstAttribute="bottom" secondItem="iDJ-b0-uIo" secondAttribute="bottom" constant="8" id="3aL-lE-KFS"/>
-                                                    <constraint firstItem="iDJ-b0-uIo" firstAttribute="top" secondItem="f0e-3N-1ay" secondAttribute="top" constant="8" id="CXI-N8-9tp"/>
-                                                    <constraint firstAttribute="trailing" secondItem="iDJ-b0-uIo" secondAttribute="trailing" constant="8" id="QuG-rB-fOq"/>
-                                                </constraints>
-                                            </view>
-                                        </subviews>
-                                        <color key="backgroundColor" red="0.97254901959999995" green="0.97254901959999995" blue="0.97254901959999995" alpha="1" colorSpace="calibratedRGB"/>
+                                    <containerView opaque="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="D00-lg-Ft4">
+                                        <rect key="frame" x="0.0" y="0.0" width="375" height="300"/>
                                         <constraints>
-                                            <constraint firstItem="f0e-3N-1ay" firstAttribute="leading" secondItem="13U-3h-0zt" secondAttribute="leading" id="AyO-qH-Zrh"/>
-                                            <constraint firstItem="f0e-3N-1ay" firstAttribute="width" secondItem="13U-3h-0zt" secondAttribute="width" id="EYC-tF-SQB"/>
-                                            <constraint firstAttribute="trailing" secondItem="f0e-3N-1ay" secondAttribute="trailing" id="OaD-rE-WNQ"/>
-                                            <constraint firstAttribute="bottom" secondItem="f0e-3N-1ay" secondAttribute="bottom" id="Uuc-kp-abY"/>
-                                            <constraint firstAttribute="height" constant="151" id="Z4H-mT-Fo4"/>
-                                            <constraint firstAttribute="height" constant="233" id="er3-ni-T2e"/>
-                                            <constraint firstAttribute="width" constant="250" id="sm8-7f-Fjg"/>
-                                            <constraint firstItem="f0e-3N-1ay" firstAttribute="top" secondItem="13U-3h-0zt" secondAttribute="top" id="ysb-Np-CVo"/>
+                                            <constraint firstAttribute="height" constant="300" id="NX2-ML-r3F"/>
```

Comment:
This constraint should be removed. It generates an unsatisfiable constraints warning in compact height environments and wouldn't be needed if the distribution is changed.